### DISCUSSION TITLE
WLAN Pi R4 with Waveshare 1.3" IPS 240x240 Display Hat

### DISCUSSION THREAD

**Discussion by tstalzer** [WLAN-Pi/feedback#58] [Hardware - WLAN Pi R4] [answered]
Hello
Being more a windows guy, I am a bit lost in the Raspberry pi World.
I am trying to setup a WLAN PI R4 with the Waveshare 1.3" IPS Display with 240x240 pixels. When the Pi starts up, it shows the Display in the upper right quadrant of the Display thus "only" using 128x128 pixels.
How can I setup the Pi correctly to use 240x240 pixels on the Display?
Everything else is working.
--Thomas

**Comment by jiribrejcha** [comment]
I don't have a 240x240 display to test it, but try this:

1. Change lines 40 and 41 in this file ```/opt/wlanpi-fpms/lib/python3.9/site-packages/fpms/modules/wlanpi_oled.py```
2. Restart the wlanpi-fpms service by ```sudo systemctl restart wlanpi-fpms``` 

<img width="427" alt="image" src="https://github.com/WLAN-Pi/feedback/assets/18701525/12e0fc54-e0fe-4bf4-b34d-3ff7eb3a7fdb">

**Comment by tstalzer** [comment, marked as answer]
Hello Jiri

Thanks for coming back to me

No, that was not enough.

I also tried to adapt constants.py. However - did not help either

I got a 128x128 display now - works perfectly.

Thanks

--Thomas
